pkg/registry/core/persistentvolume/strategy.go

```diff
@@ -97,6 +98,7 @@ func (persistentvolumeStrategy) PrepareForUpdate(ctx context.Context, obj, old r
 	newPv := obj.(*api.PersistentVolume)
 	oldPv := old.(*api.PersistentVolume)
 	newPv.Status = oldPv.Status
+	pvutil.DropDisabledSpecFields(&newPv.Spec, &oldPv.Spec)
```

Comment:
Fix a bug where pv is created or updated with a vac when feature-gate is disabled.

cc @msau42